weblate/checks/chars.py

```diff
@@ -327,10 +327,10 @@ class EscapedNewlineCountingCheck(CountingCheck):
 
 
 class NewLineCountCheck(CountingCheck):
-    r"""Check whether there is same amount of new lines"""
+    r"""Check whether the amount of newlines matches"""
     string = '\n'
     check_id = 'newline-count'
-    name = _('Newline charater count mismatch')
+    name = _('Mismatching newline character count')
```

Comment:
I was thinking about calling this rather "Mismatching lines count". What do you think?

Reply:
It is a bit hard to decipher. "Mismatching line-count" would do.
Is it always technically correct vs. number of newline chars?

Reply:
It just counts newline chars. When translating to Czech I realized that it's easier to understand when talking about number of lines than number of newline chars...